client/a8c-for-agencies/sections/overview/sidebar/contact-support/style.scss

```diff
@@ -0,0 +1,24 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
+.overview__contact-support-button {
+	width: 100%;
+}
+
+// Zendesk Chat Button
+iframe#launcher {
+	right: 24px !important;
+
+	@include break-wide() {
+		bottom: 24px !important;
+	}
+}
+
+// Zendesk Chat Window
+iframe[title="Messaging window"] {
+	margin-right: 12px;
+
+	@include break-wide() {
+		margin-bottom: 8px;
+	}
+}
```

Comment:
Customizing the messenger styles/position is pretty absent from the Zendesk documentation... using the selectors available here to ensure the chat doesn't clash with the accent border that wraps the A4A screens.